webroot/src/models/LicenseeUser/LicenseeUser.model.ts

```diff
@@ -0,0 +1,59 @@
+//
+//  user.model.ts
+//  InspiringApps modules
+//
+//  Created by InspiringApps on 4/12/20.
+//
+
+/* eslint-disable max-classes-per-file */
+import { AuthTypes } from '@/app.config';
+import deleteUndefinedProperties from '@models/_helpers';
+import { Licensee, LicenseeSerializer } from '@models/Licensee/Licensee.model';
+import { User, InterfaceUserCreate } from '@models/User/User.model';
+
+// ========================================================
+// =                       Interface                      =
+// ========================================================
+export interface InterfaceLicenseeUserCreate extends InterfaceUserCreate{
```

Comment:
Surprised none of the auto checks caught this `InterfaceUserCreate{`. I would expect eslint to catch it. We can give a quick peek if our linter rules need updating anywhere.